app/src/main/java/com/example/simplenavigation/MainActivity.java

```diff
@@ -112,8 +142,7 @@ public void onRobotReady(boolean isReady) {
     }
 
     /**
-     * Used to display robot's current position in TextView
-     * @param position holds values for X,Y position and rotation & display tilt angle
+     * @param position is used to display robot's current X,Y position and rotation & display tilt angle in TextView
```

Comment:
Missing description

Reply:
Comment changed to:
`/**
     * Function used to display robot's current X,Y position and rotation & display tilt angle in TextView
     * @param position holds values of robot's current position (X,Y,Yaw,tilt)
     */`